dsalgo/math.py

```diff
@@ -12,186 +12,198 @@
 #hcf_k_elements(l,k) - HCF of k elements in a list
 #lcm_k_elements(l, k) - LCM of k elements in a list
 
+class dmath():
```

Comment:
reason for naming it dmath?

Reply:
@ravi5175 please use function_names that explains what that function is for

Reply:
wait there's something wrong,let me correct

Reply:
@codesankalp reason for using dmath is reduce confusion among actual python math module and dsalgo math module.

Reply:
@devkapilbansal there was some branching issue thats why my file was not updated and function name with discription was not present, i have solved the issue, function description is now available with individual doc strings added to each functions